src/components/radio/daikin-radio.ts

```diff
@@ -103,13 +106,19 @@ export class DaikinRadio extends LitElement {
   @property({ type: Boolean, reflect: true })
   disabled = false;
 
+  /**
+   * Whether the radio button can be focused.
+   */
+  @property({ type: Boolean, reflect: true })
+  skipTab = false;
+
   static readonly formAssociated = true;
 
   // Define internals to let the radio button can be used in a form.
-  private _internals = this.attachInternals();
+  public internals = this.attachInternals();
```

Comment:
Internals should not be public.

```suggestion
  private _internals = this.attachInternals();
```